pages/components/rating/rating.css

```diff
@@ -0,0 +1,48 @@
+
+.rating-star-icon{
+    color: rgba(0, 0, 0, 0.26);
+}
+
+.rating-star-input:enabled .rating-star-icon{
+    cursor: pointer;
+}
+
+.rating-star-input:enabled + .rating-star-icon:hover{
+    color: #ffd675;
+}
+
+.rating-star-input{
+    width: 0;
+    margin: 0;
+    height: 0;
+    opacity: 0;
+}
+
+.rating-star-input:checked + .rating-star-icon{
+    color: #ffb400;
+}
+
+.rating-badge{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: fit-content;
+    font-size: 0.75rem;
+    line-height: 0.75rem;
```

Comment:
use px

Reply:
I have used px if the value is below 10px or if I want the size to remain constant irrespective of the parent's / root size.